### PR TITLE
[production/RRFS.v1] Update MYNN PBL & Smoke for RRFS.v1

### PR DESCRIPTION
**Pull request by haiqinli**
1. Update the MYNN PBL scheme for the RRFS.v1 code freeze.
2. Update the Smoke modules for the RRFS.v1.
3. Include the PR https://github.com/ufs-community/ccpp-physics/pull/186.

### CODE REVIEW

**Review by grantfirl — APPROVED** [review]

**Review by dustinswales — APPROVED** [review]

**Review by Qingfu-Liu — APPROVED** [review]

**Review by mzhangw — APPROVED** [review]

**Comment by grantfirl** [comment]
@haiqinli Please merge in the latest production/RRFS.v1 branch.

File: physics/smoke_dust/rrfs_smoke_wrapper.F90 [hunk 1]
```diff
@@ -1,18 +1,19 @@
-!>\file rrfs_smoke_wrapper.F90
+!>\file rrfs_smoke_wrapper.F90hwp_method
```
Comment (grantfirl) [inline physics/smoke_dust/rrfs_smoke_wrapper.F90]: I'm guessing that adding `hwp_method` is a mistake in this Doxygen comment?

**Comment by grantfirl** [comment]
@haiqinli What are your thoughts on combining https://github.com/ufs-community/ccpp-physics/pull/186 into this? Also, are you going to recreate the fv3atm PR?

**Comment by haiqinli** [comment]
@grantfirl Yes, it is fine with me that combining https://github.com/ufs-community/ccpp-physics/pull/186 into this.  Yes, I will recreate the fv3atm PR when the regression test is done. If we decide to combine https://github.com/ufs-community/ccpp-physics/pull/186 into this, I will merge it and redo the regression test.

**Comment by grantfirl** [comment]
> @grantfirl Yes, it is fine with me that combining #186 into this. Yes, I will recreate the fv3atm PR when the regression test is done. If we decide to combine #186 into this, I will merge it and redo the regression test.

OK, let's do that. Please let me know if you need help. Also, when you create the fv3atm PR, be sure to include the changes in https://github.com/NOAA-EMC/fv3atm/pull/801.

**Comment by haiqinli** [comment]
@grantfirl Sure, I am starting to merge Jili's PR, and will let you know if I need your help. Thanks for coordinating these efforts.

File: physics/smoke_dust/rrfs_smoke_wrapper.F90 (hunk 1 above)
Comment (haiqinli) [reply physics/smoke_dust/rrfs_smoke_wrapper.F90]: @grantfirl Yes, it is a mistake. Fixed. Thanks.

**Comment by grantfirl** [comment]
@haiqinli Merge looks good. I'll close #186.

**Comment by haiqinli** [comment]
@grantfirl Yes, https://github.com/ufs-community/ccpp-physics/pull/186 has been merged. Thanks.

**Comment by jkbk2004** [comment]
@MatthewPyle-NOAA @grantfirl @dustinswales tests are done at https://github.com/ufs-community/ufs-weather-model/pull/2170. can you merge this pr?

**Pull request merged**